src/engine/source/api/src/catalog/catalog.cpp

```diff
@@ -306,7 +306,7 @@ std::optional<base::Error> Catalog::putResource(const Resource& item,
     if (contentName != item.m_name)
     {
         return base::Error {
-            fmt::format("Invalid content name \"{}\" of \"{}\" for type \"{}\"",
+            fmt::format(R"(Invalid content name "{}" of "{}" for type "{}")",
```

Comment:
[Coding style, Modern language features](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices)